src/officehours_api/models.py

```diff
@@ -140,3 +171,6 @@ def post_save_user_signal_handler(sender, instance: User, created, **kwargs):
         instance.profile
     except User.profile.RelatedObjectDoesNotExist:
         instance.profile = Profile.objects.create(user=instance)
+
+
+import officehours_api.notifications
```

Comment:
This makes sure the signals in `notifications` are registered whenever the models are loaded. It's effectively similar to pasting the content of `notifications` to the end of this file, which would be an alternative, but I liked this more organizationally. Another alternative would be to keep only the signals registrations here, which would reference imports from `notifications`, but since they felt tightly coupled I kept them in the same place. Let me know if you find a better alternative!